spark/v3.5/spark-runtime/LICENSE

```diff
@@ -519,30 +419,6 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Carrot Search Labs HPPC.
```

Comment:
I don't find carrot search in the spark-runtime jar.

Reply:
I was trying to figure out how this gets a reference, it looks like Antlr Runtine (4) has this but we only have antlr-ruhntinme 3.5 so It seems like a mistake?

Reply:
At some point this was present in the Jar (when I last did a check), so it is probably a dependency change.

Reply:
Probably a dependency change.